lib/fun_with_flags/store/persistent/redis.ex

```diff
@@ -19,6 +19,8 @@ defmodule FunWithFlags.Store.Persistent.Redis do
     conf = case Config.redis_config do
       uri when is_binary(uri) ->
         {uri, @conn_options}
```

Comment:
❓ I couldn't fine this line covered by the test suite (though admittedly I had some difficulty running the entire suite). As such, I didn't provide a test for the new case below. If you can point me to the relevant test, I'm happy to add coverage for that.